.github/workflows/latest-version.yaml

```diff
@@ -9,8 +9,8 @@ jobs:
     steps:
       - name: Setup tf-latest-version
         env:
-          TF_LATEST_VERSION_VERSION: "0.0.8"
-          TF_LATEST_VERSION_SHA: "61cd892a03d970d7f35c6bf9a57a557079e455835d9a4d272a9b077b0ec5c608"
+          TF_LATEST_VERSION_VERSION: "0.0.9"
+          TF_LATEST_VERSION_SHA: "ae35d89462460d4e670734e1292c353bac6cea2932089bd891128890902f7b27"
```

Comment:
Seems correct based on the [checksums.txt](https://github-releases.githubusercontent.com/338032812/9ea03780-e0b3-11eb-9212-494dfa667303?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJYAX4CSVEH53A%2F20210709%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20210709T223826Z&X-Amz-Expires=300&X-Amz-Signature=167ae59645457cab58798a230aad4ead74b1e7d6b8845f04c3a3fd4145a2edfd&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=338032812&response-content-disposition=attachment%3B%20filename%3Dchecksums.txt&response-content-type=application%2Foctet-stream):

`ae35d89462460d4e670734e1292c353bac6cea2932089bd891128890902f7b27  tf-latest-version_0.0.9_linux_amd64.tar.gz`